crates/router/src/core/admin.rs

```diff
@@ -3986,55 +4004,96 @@ impl BusinessProfileUpdateBridge for api::BusinessProfileUpdate {
     }
 }
 
+#[cfg(all(feature = "olap", feature = "v1"))]
+pub async fn update_profile(
+    state: SessionState,
+    profile_id: &id_type::ProfileId,
+    key_store: domain::MerchantKeyStore,
+    request: api::ProfileUpdate,
+) -> RouterResponse<api::ProfileResponse> {
+    let db = state.store.as_ref();
+    let key_manager_state = &(&state).into();
+
+    let business_profile = db
+        .find_business_profile_by_profile_id(key_manager_state, &key_store, profile_id)
+        .await
```

Comment:
What about the variable `business_profile` here, and the method `find_business_profile_by_profile_id()`?

Reply:
we are taking the function name changes in upcoming pull request